vertical-pod-autoscaler/pkg/utils/metrics/quality/quality.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package quality (aka metrics_quality) - code for VPA quality metrics
+package quality
+
+import (
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/utils/metrics"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	metricsNamespace = metrics.TopMetricsNamespace + "quality"
+)
+
+var (
+	usageRecommendationRelativeDiff = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: metricsNamespace,
+			Name:      "usage_recommendation_relative_diffs",
+			Help:      "Diffs between usage and recommendation, normalized by recommendation value",
+			Buckets: []float64{-1., -.75, -.5, -.25, -.1, -.05, -.005, -.001,
```

Comment:
Please add -.01 for symmetry.
I also wonder about adding +-0.002 & +-0.02, similar to .25 or even more detailed buckets; WDYT?

Reply:
Added .01
As for more detailed buckets in the < 0 part, I'm not sure it is that interesting, I'm more invested in the usage > recommendation part, but I can add more.

Reply:
I would keep the buckets symmetric (at least as far as they can be kept that way).
What I meant by added 0.002 and 0.02 boundaries was to get more precise information, jump from 1 to 5 seems big.

Reply:
Got it now. Done.